#! /bin/sh /usr/share/dpatch/dpatch-run
## manpages.dpatch by Stephane Glondu <steph@glondu.net>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix errors in ocamlc and ocamlopt manpages (see Mantis #4629)


@DPATCH@
diff --git a/man/ocamlc.m b/man/ocamlc.m
index 0ce1e8d..b1fb8bf 100644
--- a/man/ocamlc.m
+++ b/man/ocamlc.m
@@ -421,8 +421,8 @@ as a preprocessor for each source file. The output of
 is redirected to
 an intermediate file, which is compiled. If there are no compilation
 errors, the intermediate file is deleted afterwards. The name of this
-file is built from the basename of the source file with the extension
-.ppi for an interface (.mli) file and .ppo for an implementation
+file is built from the basename of the source file with the extension .ppi
+for an interface (.mli) file and .ppo for an implementation
 (.ml) file.
 .TP
 .B \-principal
@@ -561,7 +561,7 @@ into errors.  The compiler will stop with an error when one of these
 warnings is emitted.  The
 .I warning\-list
 has the same meaning as for
-the "-w" option: an uppercase character turns the corresponding
+the "\-w" option: an uppercase character turns the corresponding
 warning into an error, a lowercase character leaves it as a warning.
 The default setting is
 .B \-warn\-error\ a
diff --git a/man/ocamlopt.m b/man/ocamlopt.m
index 3872bd8..2a44836 100644
--- a/man/ocamlopt.m
+++ b/man/ocamlopt.m
@@ -222,7 +222,7 @@ and edit that file to remove all declarations of unexported names.
 Add the given directory to the list of directories searched for
 compiled interface files (.cmi) and compiled object code files
 (.cmo). By default, the current directory is searched first, then the
-standard library directory. Directories added with -I are searched
+standard library directory. Directories added with \-I are searched
 after the current directory, in the order in which they were given on
 the command line, but before the standard library directory.
 
@@ -536,7 +536,7 @@ into errors.  The compiler will stop with an error when one of these
 warnings is emitted.  The
 .I warning\-list
 has the same meaning as for
-the "-w" option: an uppercase character turns the corresponding
+the "\-w" option: an uppercase character turns the corresponding
 warning into an error, a lowercase character leaves it as a warning.
 The default setting is
 .B \-warn\-error\ a
@@ -577,7 +577,7 @@ trigonometric operations
 .BR cos ,
 .BR sin ,
 .B tan
-have their range reduced to [-2^64, 2^64].
+have their range reduced to [\-2^64, 2^64].
 
 .SH OPTIONS FOR THE AMD64 ARCHITECTURE
 
